File: 1635-number-of-good-pairs/number-of-good-pairs.cpp
class Solution {
public:
    int numIdenticalPairs(vector<int>& nums) {
      int n=nums.size();
      int cnt=0;
      for(int i=0;i<n-1;i++)
      {
          int j=i+1;
          while(j<n)
          {
              if(nums[i]==nums[j])
              {
                  cnt++;
              }
              j++;
          }
      }
      return cnt;
    }
};

File: 1635-number-of-good-pairs/number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int i,j;
        int cnt=0;
        for(i=0;i<nums.length-1;i++)
        {
            for(j=i+1;j<nums.length;j++)
            {
                if(nums[i]==nums[j]) cnt++;
            }
        }
        return cnt;
    }

}